README.md

```diff
@@ -4,70 +4,78 @@
 
-#### About this prototype
+## Overview
 
-The prototype in this repository uses [Swagger](http://swagger.io/) to specify the API in [dss-api.yml](dss-api.yml),
-and [Connexion](https://github.com/zalando/connexion) to map the API specification to its implementation in Python.
+The DSS is a replicated data storage system designed for hosting large sets of scientific experimental data on 
+[Amazon S3](https://aws.amazon.com/s3/) and [Google Storage](https://cloud.google.com/storage/). The DSS exposes an API
+for interacting with the data and is built using [Chalice](https://github.com/aws/chalice),
+[API Gateway](https://aws.amazon.com/api-gateway/) and [AWS Lambda](https://aws.amazon.com/lambda/). The API also
+implements [Step Functions](https://aws.amazon.com/step-functions/) to orchestrate Lambdas for long-running tasks such
+as large file writes. You can find the API documentation and give it a try [here](https://dss.data.humancellatlas.org/).
 
-You can use the
-[Swagger Editor](http://editor.swagger.io/#/?import=https://raw.githubusercontent.com/HumanCellAtlas/data-store/master/dss-api.yml) to
-review and edit the prototype API specification. When the prototype app is running, the Swagger spec is also available
-at `/v1/swagger.json`.
-
-The prototype is deployed continuously from the `master` branch, with the resulting producer and consumer API available
-at https://dss.integration.data.humancellatlas.org/.
-
-#### Installing dependencies for development on the prototype
+<!-- ![Architecture Diagram](Group%202.png) -->
 
-The HCA DSS prototype development environment requires Python 3.6+ to run. Run `pip install -r requirements-dev.txt` in
-this directory.
+#### DSS API
+The DSS API uses [Swagger](http://swagger.io/) to define the [API specification](dss-api.yml) according to the
+[OpenAPI 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
+[Connexion](https://github.com/zalando/connexion) is used to map the API specification to its implementation in Python.
 
-#### Installing dependencies for the prototype
-
-The HCA DSS prototype requires Python 3.6+ to run. Run `pip install -r requirements.txt` in this directory.
+You can use the
+[Swagger Editor](http://editor.swagger.io/#/?import=https://raw.githubusercontent.com/HumanCellAtlas/data-store/master/dss-api.yml)
+to review and edit the API specification. When the API is live, the spec is also available at `/v1/swagger.json`.
 
-#### Pull sample data bundles
+## Getting Started
+In this section, you'll configure and deploy a local API server and your own suite of cloud services to run a
+development version of the DSS.
 
-Tests also use data from the data-bundle-examples subrepository. Run: `git submodule update --init`
+### Install Dependencies
+Python 2.7 or Python 3.6 is required to run the DSS. The DSS supports Python versions supported by AWS Lambda which can
```

Comment:
I think mentions of Python 2.7 should be dropped here (we don't support Python 2.7 on the server, and this section doesn't seem to apply to the client).